src/ophys_etl/modules/decrosstalk/__main__.py

```diff
@@ -13,6 +17,11 @@ class DecrosstalkWrapper(argschema.ArgSchemaParser):
     default_output_schema = decrosstalk_schema.DecrosstalkOutputSchema
 
     def run(self):
+        ophys_etl_commit_sha = os.environ.get("OPHYS_ETL_COMMIT_SHA",
+                                              "local build")
+
+        self.logger.info(f"OPHY_ETL_COMMIT_SHA: {ophys_etl_commit_sha}")
```

Comment:
you might want to default your input schema to log_level INFO unless you're relying on the LIMS strategy to be doing that explicitly.

Reply:
Done.